src/flint/fmpz_mat.jl

```diff
@@ -1554,7 +1554,11 @@ function _solve_dixon(a::ZZMatrix, b::ZZMatrix)
 end
 
 #XU = B. only the upper triangular part of U is used
-function _solve_triu_left(U::ZZMatrix, b::ZZMatrix)
+function AbstractAlgebra._solve_triu_left(U::ZZMatrix, b::ZZMatrix; side::Symbol = :left)
```

Comment:
There is no `AbstractAlgebra._solve_triu_left`
```suggestion
function _solve_triu_left(U::ZZMatrix, b::ZZMatrix; side::Symbol = :left)
```